db/migrate/20150618183029_add_extensions_to_versions.rb

```diff
@@ -0,0 +1,5 @@
+class AddExtensionsToVersions < ActiveRecord::Migration
+  def change
+    add_column :versions, :extensions, :boolean
```

Comment:
Add a default state here or we'll end up with a tri-state boolean column (true, false, nil)